src/boot/mexpr.ml

```diff
@@ -438,28 +448,28 @@ let rec val_equal v1 v2 =
   | _ -> false
 
 
-(* Convert a term into de Bruijn indices. *)
-let rec debruijn env t =
-  let rec find fi env n x =
-    (match env with
-     | VarTm(y)::ee -> if y =. x then n else find fi ee (n+1) x
-     | [] -> raise_error fi ("Unknown variable '" ^ Ustring.to_utf8 x ^ "'")) in
+(* Add symbol associations between lambdas, patterns, and variables *)
+let rec symbolize env t =
+  let findsym fi x kind env = try List.assoc x env
+    with Not_found -> raise_error fi ("Unknown " ^ kind ^ " '" ^ Ustring.to_utf8 x ^ "'")
+  in
   let rec dbPat env = function
```

Comment:
The name `dbPat` stems from de bruijn indices, should perhaps be renamed now that we're removing them (I'll probably do that during my changes if it's not done here).

Reply:
OK, good. Changed to sPat.